CustomerReviewsModule.Web/Scripts/blades/review-detail.tpl.html

```diff
@@ -1,26 +1,31 @@
-﻿<div class="blade-content __medium-wide">
+﻿<div></div>
+<div class="blade-content __medium-wide">
     <div class="blade-inner">
         <div class="inner-block">
             <form class="form" name="detailForm">
                 <div class="form-group clearfix">
                     <div class="column">
                         <div class="form-group">
-                            <label class="form-label">{{ 'customerReviewsModule.blades.review-detail.labels.productId' | translate }}</label>
+                            <label class="form-label">{{ 'customerReviewsModule.blades.review-detail.labels.authorNickname' | translate }}</label>
```

Comment:
productId is an important field, it should be shown

Reply:
fixed